tests/test_model.py

```diff
@@ -596,6 +596,7 @@ def test_against_original_gemma(model_name, device, dtype):
         rope_theta=ours_config.rope_base,
         attention_bias=ours_config.bias,
         tie_word_embeddings=True,
+        hidden_act="gelu_pytorch_tanh",
```

Comment:
This solved:

> I looked through the code maybe a dozen of times and could find anything wrong. Still don't understand why when we convert from HF to Lit the test passes, but when from Lit to HF - doesn't. If in the single test I first convert HF -> Lit and then right after Lit -> HF - test passes 🤷.

?

Reply:
Yes.
I didn't add any `pytest.xfail`, only forced to use this correct approximation for the activation function.
All tests passed.

Reply:
But I haven't tried to generate anything with the added approximation.
Here we rely on Sebastian.
I would recommend merging only after @rasbt gives us the green light.